chromium_src/chrome/browser/ui/views/tabs/tab.cc

```diff
@@ -17,6 +19,27 @@
 #define BRAVE_UI_VIEWS_TABS_TAB_UPDATE_ICON_VISIBILITY \
   showing_close_button_ &= mouse_hovered();
 
+#define GetWidthOfLargestSelectableRegion \
+  GetWidthOfLargestSelectableRegion_ChromiumImpl
 #include "src/chrome/browser/ui/views/tabs/tab.cc"
 #undef BRAVE_UI_VIEWS_TABS_TAB_UPDATE_ICON_VISIBILITY
 #undef BRAVE_UI_VIEWS_TABS_TAB_ALERT_INDICATOR_POSITION
+#undef GetWidthOfLargestSelectableRegion
```

Comment:
this line shoud go before `#undef BRAVE_UI_VIEWS_TABS_TAB_UPDATE_ICON_VISIBILITY`

Reply:
Fixed.